

#Coordinates of 4 corners of the detected ROI
#float32 x
#float32 y
#float32 h
#float32 w

#lat lon of the detected bbox
float32 lat
float32 lon

float32[] bbox

float64 confidence # score of the detection

string object_class #detected class

